src/main/java/com/rapid7/client/dcerpc/msvcctl/ServiceControlManagerService.java

```diff
@@ -79,12 +90,15 @@ public IServiceStatusInfo stopService(String service, ContextHandle serviceManag
         } else throw new RPCException("StopService", response.getReturnValue());
     }
 
-    public boolean changeServiceConfig(String service, ContextHandle serviceManagerHandle, ServiceConfigInfo serviceConfigInfo)
-            throws IOException {
+    public boolean changeServiceConfig(String service,
+                                       ContextHandle serviceManagerHandle,
+                                       IServiceConfigInfo serviceConfigInfo)
+        throws IOException {
         ContextHandle serviceHandle = getServiceHandle(service, serviceManagerHandle);
         RChangeServiceConfigWRequest request = new RChangeServiceConfigWRequest(serviceHandle, serviceConfigInfo);
         RChangeServiceConfigWResponse response = transport.call(request);
-        return SystemErrorCode.ERROR_SUCCESS.is(response.getReturnValue());
+        if(SystemErrorCode.ERROR_SUCCESS.is(response.getReturnValue())) return true;
```

Comment:
I personally prefer the following format.
`
if (!SystemErrorCode.ERROR_SUCCESS.is(response.getReturnValue())
    throws new RPCException("ChangeServiceConfig", response.getReturnValue());
return true;
`